packages/webapp/src/elections/api/fixtures.ts

```diff
@@ -87,7 +87,7 @@ export const fixtureVoteDataRows: VoteData[] = [
         member: "egeon.edev",
         round: 1,
         index: 5,
-        candidate: "edenmember13",
```

Comment:
Do you want to leave this scenario in the fixture? fine if so; just wanted to ensure you intended to alter the fixture in this PR

Reply:
Yeah, the fixture had `egeon.edev` voting for someone outside their round.